solver/src/solver/http_solver.rs

```diff
@@ -507,6 +514,17 @@ mod tests {
             });
         let mock_token_info_fetcher: Arc<dyn TokenInfoFetching> = Arc::new(mock_token_info_fetcher);
 
+        let mut mock_buffer_retriever = MockBufferRetrieving::new();
+        mock_buffer_retriever
+            .expect_get_buffers()
+            .return_once(move |_| {
+                hashmap! {
+                    buy_token => U256::from(42),
+                    sell_token => U256::from(1337),
+                }
+            });
+        let mock_buffer_retriever: Arc<dyn BufferRetrieving> = Arc::new(mock_buffer_retriever);
```

Comment:
Would it not make more sense to test get buffers as a standalone thing directly in file defining get_buffers?

Reply:
Done!